src/components/Create/components/pages/FundingCycles/hooks/FundingCyclesForm.ts

```diff
@@ -71,5 +87,22 @@ export const useFundingCyclesForm = () => {
     }
   }, [selection, duration, dispatch])
 
+  useEffect(() => {
+    if (launchDate === undefined) return
+    if (launchDate === null) {
+      dispatch(
+        editingV2ProjectActions.setMustStartAtOrAfter(
+          DEFAULT_MUST_START_AT_OR_AFTER,
+        ),
+      )
+      return
+    }
+    dispatch(
+      editingV2ProjectActions.setMustStartAtOrAfter(
+        launchDate?.unix().toString() ?? null,
```

Comment:
**suggestion** is it still worth falling back to `DEFAULT_MUST_START_AT_OR_AFTER` here?

Reply:
I think I put this in as a safeguard when I was testing some unexpected behaviour. I have removed this up into the other line in case somethign unexpected was here